charts/rawls/templates/migration/job.yaml

```
@@ -0,0 +1,88 @@
+{{- if .Values.migration.enabled }}
+{{- $imageTag := .Values.migration.imageTag | default .Values.global.applicationVersion -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: rawls-liquibase-migration
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocr.argoproj.io/hook-delete-policy: BeforeHookCreation
+  labels: 
+    {{- include "rawls.labels" . | nindent 4 }}
```

I realize rawls doesn't do this in other places but when you use `nindent` you can actually define labels like this. It reads a little cleaner this way imo.
```suggestion
  labels: {{- include "rawls.labels" . | nindent 4 }}
```